app/models/lookup_keys/puppetclass_lookup_key.rb

```diff
@@ -40,4 +41,8 @@ def validate_default_value
   def puppet?
     true
   end
+
+  def check_override_selected
+    errors.add(:override, _("must be true to edit the parameter")) unless self.override || (self.changes.size == 1 && self.changes["description"].present?) || self.new_record?
```

Comment:
@dLobatog I'll change to an `if` but I can't remove the middle part about description since it can be edited at any time  even when override is false and this is an existing record.